LibSWBF2/Types/Vector2.h

```diff
@@ -26,5 +26,5 @@ namespace LibSWBF2::Types
 		float_t m_Y = 0.0f;
 
 		Types::String ToString();
-	};
+	} __attribute__((aligned(4), packed));
```

Comment:
As mentioned before, this allows for quick/clean marshaling.  Not tested on Windows though.  I'm not even sure it's necessary, the compiler may create this layout by default, but I couldn't find a definitive answer.